addons/im_livechat/static/src/embed/common/thread_model_patch.js

```diff
@@ -34,17 +34,15 @@ patch(Thread, {
             if (this.env.services["im_livechat.chatbot"].isChatbotThread(thread)) {
                 thread.chatbotTypingMessage = {
                     id: this.env.services["mail.message"].getNextTemporaryId(),
-                    res_id: thread.id,
-                    model: thread.model,
```

Comment:
code should still work the same by inserting `res_id` and `model`, all this diff should be reverted